Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:lts-hydrogen
+
+WORKDIR /usr/app
+COPY package.json .
+COPY yarn.lock .
+COPY . .
+
+RUN yarn
```

Comment:
I moved `yarn run dev` into the Dockerfile and tried to build an image. The image never finishes building because `yarn run dev` starts the dev server, which runs forever if never interrupted.